storm-client/src/jvm/org/apache/storm/messaging/netty/Context.java

```diff
@@ -65,20 +65,8 @@ public synchronized IConnection bind(String storm_id, int port) {
      * establish a connection to a remote server
      */
     public synchronized IConnection connect(String storm_id, String host, int port, AtomicBoolean[] remoteBpStatus) {
```

Comment:
This no longer needs to be synchronized.  That was used to protect connections, and is no longer needed if all we do is create a new Client.